CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ maintainability of worker queries, and improve DB performance. ([PR](https://git
   [our worker/controller compatiblity policy](https://developer.hashicorp.com/boundary/docs/enterprise/supported-versions#control-plane-and-worker-compatibility).
 * Add concurrency limit on the password hashing of all password auth methods.
   ([PR](https://github.com/hashicorp/boundary-plugin-aws/pull/5437)).
+* ui: Improve worker filter workflow for targets, vault credential-stores, and storage-buckets. ([PR](https://github.com/hashicorp/boundary-ui/pull/2614)).
```

Comment:
I think this ended up splitting up a changelog line regarding the concurrency limit during a merge conflict resolution.